domain/infosync/info.go

```diff
@@ -451,6 +460,95 @@ func (is *InfoSyncer) updateTopologyAliveness(ctx context.Context) error {
 		clientv3.WithLease(is.topologySession.Lease()))
 }
 
+// GetPrometheusAddr gets prometheus Address
+func GetPrometheusAddr() (string, error) {
+	is, err := getGlobalInfoSyncer()
+	if err != nil {
+		return "", err
+	}
+
+	// if the cache of prometheusAddr is over 5s, update the prometheusAddr
+	if time.Since(is.modifyTime) < TablePrometheusCacheExpiry {
+		return is.prometheusAddr, nil
+	}
+	is.prometheusAddr = ""
```

Comment:
Why need reset here?

Reply:
It was used in some outdated logic, now that logic have been deleted and this code can be deleted too.